sbatches/farmshare/README.md

```diff
@@ -1,3 +1,10 @@
 # Farmshare
 
-Hi friend! These haven't been tested fully yet. Do you want to help? Please work with @vsoch!
+Hi friend! These haven't been tested fully yet. Do you want to help? Please work with @vsoch! (Testing Phase)
+1. Following the original tutorial - the container to use in Rice is located at /farmshare/home/classes/bioe/301p/ce/ces - maintained by Dr.Paul Nuyujukian of Bioengineering.
+2. On Rice, type `model load singularity`, and then `singularity exec /farmshare/home/classes/bioe/301p/ce/ces jupyter notebook --generate-config`. 
+3. Choose a password and type and verify password, it will be stored and later used to access the notebooks. 
+4. Fill out the params_farmshare.sh file as the original tutorial. I have created a separate setup.sh (called setup_farmshare.sh). Choose a port that is higher than 32768 for the tunnel to work. 
```

Comment:
This file is great! I think this needs to be updated?

Reply:
Updated Readme on 
fca03b1.